### PR TITLE
Add audio interfaces to network diagram

### PR DESCRIPTION
**Pull request by Echoskope**
Added the microphone interfaces.

### CODE REVIEW

**Comment by MattIPv4** [comment]
I think the "w/ Axis P8221 I/O Audio Module" strings are probably outdated now, or are there two I/O modules for some cams?

**Comment by Echoskope** [comment]
Good point, I removed that text and clarified the Microphone block from the Audio I/O block.

**Comment by MattIPv4** [comment]
👀 One other thing, it'd be great to be consistent with either "AXIS" or "Axis" -- we have a mix of both currently?

**Comment by Echoskope** [comment]
The device description on the Axis website has as AXIS, should we keep that same style anytime we use the brand name?

**Comment by MattIPv4** [comment]
Yeah, all the products seem to use "AXIS", so that seems fine to use here

File: apps/website/src/data/network.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ type NetworkItemCore = {
 };
 
 type NetworkItemSwitch = NetworkItemCore & {
-  type: "switch" | "converter";
+  type: "switch" | "converter"  | "interface";
```
Comment (MattIPv4) [inline apps/website/src/data/network.ts:26]:
```suggestion
  type: "switch" | "converter" | "interface";
```

**Review by MattIPv4 — APPROVED** [review]
LGTM, ty

**Pull request merged**